eml/components/profileSettings/deleteAccountButton.js

```diff
@@ -0,0 +1,65 @@
+import { TouchableOpacity, Alert, View } from 'react-native'
+import { useNavigation } from '@react-navigation/native'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { deleteUser } from '../../api/userApi'
+import {clearAsyncStorage} from "../../services/StorageService";
+import React from 'react';
+import Text from '../general/Text';
+
+const LOGIN_TOKEN = '@loginToken'
+const USER_INFO = '@userInfo'
+
+export default function DeleteAccount() {
+  const navigation = useNavigation()
+
+  async function Delete() {
+    try {
+      const obj = JSON.parse(await AsyncStorage.getItem(USER_INFO))
+      
+      if (obj !== null) {
+        try {
+          await deleteUser(obj.id) // skift obj.id til users actual id to test this function
+            .then(function (response) {
+              console.log(response)
+              AsyncStorage.multiRemove([LOGIN_TOKEN, USER_INFO]).then((r) => {
+                console.log('User account deleted successfully!')
+                navigation.navigate('LoginStack')
+              })
+            })
+            .catch((error) => {
+              console.log(error)
+            })
+          await clearAsyncStorage();
+        } catch (e) {
+          console.log(e)
+        }
+      } else {
+        Alert.alert('Error', 'User not found')
+      }
+    } catch (e) {
+      console.log(e)
+    }
+  }
+
+  const deleteAlert = () =>
```

Comment:
I believe there is a component for this. Perhaps we could use that here. Also there is a leftover console.log here, I think

Reply:
We will look at reusing showAlert() instead, in a later iteration